src/components/MovieList/MovieList.js

```diff
@@ -1,32 +1,73 @@
-import React from 'react';
-import poster from '../../assets/images/inceptionPoster.png';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
 import MovieCard from '../MovieCard/MovieCard';
+import { Modal, AddMovieModal, DeleteMovieModal } from '../../modals/index';
```

Comment:
Why AddMovieModal is used in multiple places ?

Reply:
Fixed in prev PR. Created a separate EditMovieModal component.